pkg/minishift/addon/manager/addon_manager.go

```diff
@@ -59,21 +61,28 @@ func NewAddOnManager(baseDir string, configMap map[string]*addon.AddOnConfig) (*
 			continue
 		}
 
-		addOn, err := p.Parse(fullPath)
+		addOns, err := p.Parse(fullPath)
 		if err != nil {
 			_, ok := err.(parser.ParseError)
 			if ok {
-				glog.Warning(fmt.Sprintf("Skipping addon '%s' in '%s' due to parse error: %s", f.Name(), fullPath, err.Error()))
+				fmt.Println(fmt.Sprintf("Skipping addon '%s' in '%s' due to parse error: %s", f.Name(), fullPath, err.Error()))
```

Comment:
This is right now only for test purpose, I will remove it before final PR update.